html/pfappserver/lib/pfappserver/Model/Node/Tab/WMI.pm

```diff
@@ -30,12 +30,12 @@ sub process_view {
     my $mac = $c->stash->{mac};
```

Comment:
All of this will need to be adjusted in the API as well so the new admin can get these improvements